### ISSUE TITLE
Search Serialized Items from Invoice Management

### ISSUE THREAD

**Issue by SajAbu** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#992]
Currently i cannot search for an IMEI through invoice management. This means i cannot search my inventory properly. this is something that is important when searching sales made by employees and to reduce potential theft. Could you please look into this and add this option?

**Comment by Uzair-RepairDesk** [comment]
Hi @SajAbu 
Thank you for bringing it out to our attention. 
Can you please confirm if you are highlighting IMEI of trade-in devices or the IMEI field of "Mobile, Accessories & Parts inventory? 

Once you confirm, we can look into it. Thanks

**Comment by Uzair-RepairDesk** [comment]
Hi @SajAbu  - since I haven't heard back from you, I’m assuming you're all set for now, but you can always give me a shout here or reopen it if anything else comes up! Thanks

**Comment by SajAbu** [comment]
hi, its not resolved yet im afraid. its when you go to search the invoices. there is no option to search by imei. any sort of imei

**Comment by SajAbu** [comment]
Hi Uzair, what further information do you need from me to resolve this issue?

**Comment by Uzair-RepairDesk** [comment]
Hi @SajAbu 
I have reopened the post.
Can you please specify the IMEI criteria you mentioned above? Do you want to add criteria to search an invoice from a specific serial number under the manage invoices section?

**Comment by SajAbu** [comment]
yes. i want to be able to search for an imei through the invoices.

**Comment by SajAbu** [comment]
so i can see when a particular imei was repaired or sold to a customer

**Comment by Uzair-RepairDesk** [comment]
Are you using serialized inventory feature of RepairDesk and keep IMEI's as serials @SajAbu ?

**Comment by SajAbu** [comment]
yes
